domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/topic/Topic.kt

```diff
@@ -0,0 +1,17 @@
+package com.krzykrucz.elesson.currentlesson.domain.topic
+
+import arrow.core.Either
+import com.krzykrucz.elesson.currentlesson.domain.NonEmptyText
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.LessonIdentifier
+
+data class TopicTitle(val title: NonEmptyText)
+
+data class LessonTopic(val topicTitle: TopicTitle)
```

Comment:
according to Design Level Event Storming and Business Terms, shouldn't `LessonTopic` contains also `date` and some ordinal indication?

Reply:
Yup, you're right, I guess I somehow missed that :D Corrected.

Reply:
:+1: